docs/robots/warthog/user_manual_warthog.mdx

```diff
@@ -1,32 +1,70 @@
 ---
 title: User Manual
-sidebar_position: 2
+sidebar_position: 1
+toc_min_heading_level: 2
+toc_max_heading_level: 4
 ---
 
 import ComponentIntroductionWarthog from "/components/introduction_warthog.mdx";
 import ComponentNetplanSetup from "/components/netplan_setup.mdx";
+import ComponentUsingRos from "/components/using_ros.mdx";
 import Support from "/components/support.mdx";
 import ComponentCommonSafeWorkProcedures from "/components/common_safe_work_procedures.mdx";
 
+<ComponentIntroductionWarthog />
+
 ## Introduction
 
-<ComponentIntroductionWarthog />
+Clearpath Robotics Warthog is a rugged, all-terrain unmanned ground vehicle capable of brief periods of locomotion
+in water. Warthog fully supports the Robot Operating System (ROS) and can be equipped with a variety of payloads,
+including sensors and manipulators, to accommodate a wide range of robotics applications in mining, agriculture
+and environmental monitoring.
 
-### What's Included
+### Shipment Contents
 
-Each Warthog includes:
+Your Warthog shipment contains the following:
 
 - Warthog UGV
   - Onboard computer
-  - User Panel with power, Ethernet, Serial(RS232) and USB connectivity
-  - 48V Lead-Acid Battery Pack
+  - User Breakout Panel with power, Ethernet, Serial (RS232) and USB connectivity
+  - 48 V Lead-Acid Battery Pack or 48 V Lithium Battery Pack
 - One battery charger
-- One Futaba Remote Control (R/C)
+- One Futaba remote control
 - One Warthog quick start guide
 - One wireless stop remote
 
+If you purchased standard payload modules or custom integration services with Warthog, then additional equipment
+will be included per your specific configuration, plus further documentation as required.
+
 ### Hardware Overview
 
+#### System Architecture
+
+Warthog is built around an onboard computer running Ubuntu (Intel-based computer or Jetson developer kit), paired with a
+32-bit microcontroller MCU. The MCU handles IO, system and battery monitoring, and provides an interface to the CAN-controlled
+motor drivers. The communication channel between the MCU and computer is an Ethernet connection.
+
+<center>
+  <figure>
+    <img
+      src="/img/robot_images/warthog_images/robot_warthog_7.png"
+      width="800"
+    />
+    <figcaption>System Architecture</figcaption>
+  </figure>
+</center>
+
+#### System Features
+
+The Warthog is shown below and includes:
+
+- Battery charger access
+- Bilge pumps
+- User breakouts
+- Payload integration area
+- Four E-stop buttons (one on each corner)
```

Comment:
I think we should change this to 'Stop buttons'. The image shows E-stop, but I would like to start aligning our documentation with Industry standards. Boxer has a Emergency stop system, but our other robots do not.

'Stop buttons' should still be descriptive enough.

Reply:
Updated throughout the user manual, including the image